content/master/concepts/usages.md

```diff
@@ -189,6 +189,41 @@ spec:
           baz: qux
 ```
 
+### Replay blocked deletion attempt
+
+By default, the deletion of a `Usage` resource doesn't trigger the deletion of
+the resource in use even if there were deletion attempts blocked by the `Usage`.
+Replaying the blocked deletion is possible by setting the
+{{<hover label="replay" line="6">}}replayDeletion{{</hover>}} field to `true`.
+
+```yaml {label="replay"}
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: Usage
+metadata:
+  name: release-uses-cluster
+spec:
+  replayDeletion: true
+  of:
+    apiVersion: eks.upbound.io/v1beta1
+    kind: Cluster
+    resourceRef:
+      name: my-cluster
+  by:
+    apiVersion: helm.crossplane.io/v1beta1
+    kind: Release
+    resourceRef:
+      name: my-prometheus-chart
+```
+
+{{<hint "tip" >}}
+
+Replay deletion is useful when the used resource is part of a composition.
+This configuration radically decreases time for the deletion of the used
```

Comment:
Why isn't it on by default?

Reply:
Because it is a behavioral change that I didn't feel comfortable to start by enabling. We will reconsider while bumping the API to beta: https://github.com/crossplane/crossplane/pull/5394#discussion_r1518582368